src/app/layout.tsx

```diff
@@ -27,14 +29,11 @@ export default function Layout({ children }: { children: ReactNode }) {
   });
 
   return (
-    <html lang="en" className={(blenderBold.className, blenderBook.className)}>
+    <html
+      lang="en"
+      className={`${(blenderBold.className, blenderBook.className)}`}
+    >
```

Comment:
I think this is not related to this PR, and we don't will follow with that right?